covid_api/db/static/datasets/co2-diff.json

```diff
@@ -24,7 +26,7 @@
         "detections-ship",
         "detections-plane"
     ],
-    "enabled": true,
+    "enabled": false,
```

Comment:
All CO2 layers are using the 16 day product:

```
${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2-diff/xco2_16day_diff.{date}.tif&resampling_method=bilinear&bidx=1&rescale=-0.000001%2C0.000001&color_map=rdbu_r
```

Reply:
Done